### ISSUE TITLE
Automate packaging

### ISSUE THREAD

**Issue by sd2k** [sd2k/grafana-materialize-datasource#2]
Plugins need to be packaged as a .zip file containing the output of `yarn build` as well as the backend executable for (ideally) lots of targets:

```
/// Map of Go targets, as understood by Grafana, to Rust targets.
static RUST_TARGETS: Map<&'static str, &'static str> = phf_map! {
    "linux_amd64" => "x86_64-unknown-linux-musl",
    "linux_arm" => "armv7-unknown-linux-musleabihf",
    "linux_arm64" => "aarch64-unknown-linux-musl",
    "darwin_amd64" => "x86_64-apple-darwin",
    "darwin_arm64" => "aarch64-apple-darwin",
    "windows_amd64" => "x86_64-pc-windows-gnu",
};
```

As the above suggests, I have a bit of code which can automate the cross-compilation from a Mac, but it's not super generalizable to other OS's 😞 It'd be good if we could just run `cargo xtask build` to build and copy the executables into `./dist` for all architectures, similar to running `mage -v` when using the Go plugin SDK.

**Comment by jeschkies** [comment]
Part was merged with https://github.com/sd2k/grafana-materialize-datasource/pull/7

**Comment by sd2k** [comment]
Fixed by #9 which does this in CI.
